hack/testing-olm-upgrade/secrets/elasticsearch-secret.yaml

```diff
@@ -0,0 +1,13 @@
+apiVersion: v1
```

Comment:
is there a reason we're hardcoding the values of secrets here vs generating the certs and secrets at the beginning of the test?

Reply:
@igor-karpukhin to rehash on this -- we have a make target that should be able to generate certs, can we reuse that logic instead of pushing static cert files? that seems less fragile

Reply:
Fixed